code/modules/antagonists/wizard/equipment/soulstone.dm

```diff
@@ -257,25 +269,34 @@
 				switch(construct_class)
 					if("Juggernaut")
 						if(iscultist(user) || iswizard(user))
-							makeNewConstruct(/mob/living/simple_animal/hostile/construct/juggernaut, A, user, 0, T.loc)
+							if(theme == "wizard")
+								makeNewConstruct(/mob/living/simple_animal/hostile/construct/juggernaut/mystic, A, user, 0, T.loc)
```

Comment:
So a wiz can take a cult or a wiz construct shell and make a wiz juggernaut, but can't make anything with a purified construct? Seems kinda odd

Reply:
That's how it was before but it makes sense to change it.